interface/modules/zend_modules/module/Installer/src/Installer/Controller/InstallerController.php

```diff
@@ -110,6 +113,12 @@ public function registerAction()
 
     public function manageAction()
     {
+        // Pass a unique variable, so below scripts can
+        // not be run on their own
+        $unique_id = RandomGenUtils::createUniqueToken();
+        $_SESSION['lang_module_unique_id'] = $unique_id;
```

Comment:
I move this directly above where script is called (like for did for the second example below). Then it's clear what it's used for. Also change the name to something pertinent such as:
$_SESSION['acl_setup_unique_id'] = $unique_id;

Reply:
Hi @bradymiller. Fixed.